Updated anime-cli

exKitsune opened the pull request:
Added batching, resolution, no show, and fixed a bug where media player would close upon download complete

smhmd commented:
Could you add listing as well? Use case: piping into a file or a video player: `anime-cli -q "anime" -e 14 -l | mpv --prefetch-playlist --playlist=-`

exKitsune committed:
Included resolution flag

exKitsune committed:
Included resolution flag

exKitsune committed:
Included resolution flag

exKitsune committed:
Introduced batch downloading and no show

Modified the join logic so that logging in and joining the irc channel is more consistent

Fixed the media player window from closing if download finishes

exKitsune committed:
Added dir creation for organization

exKitsune committed:
Changed message read to a non-blocking method in order to have external timeouts
Added resume functionality
Attempted to add better batching functionality

exKitsune committed:
Updated readme

exKitsune committed:
Updated readme with todo list

exKitsune committed:
Cropped long filenames so pbr doesn't spam

exKitsune committed:
Fixed episode logic when not passing an epsiode number

exKitsune committed:
Fixed progress bars working during the wait
Added ability to seamlessly stream from one episode to next

exKitsune committed:
Updated readme

exKitsune committed:
Added a check to make sure it is a valid media file
Added the ability to compile without mpv
Added the ability to stream a single file using default media players
Added the ability to be ran without args

exKitsune committed:
Updated readme and tips

exKitsune committed:
Fixed defaults in "gui" mode

exKitsune committed:
Fixed defaults in "gui" mode

exKitsune committed:
Updated readme

exKitsune committed:
Added comments, trimmed filename even more

exKitsune committed:
Trimmed filename even more by adding case for super small screens

exKitsune committed:
Trimmed status bar messages as well
Added an ability to browse local anime collection, and the ability to organize it as well

exKitsune committed:
Cleaned up progress bar codes

exKitsune committed:
Changed strategy of low width terminals, will now remove bars altogether

exKitsune committed:
Updated Readme and toml

exKitsune committed:
Update Cargo.toml

exKitsune committed:
Updated Readme and toml
